### DISCUSSION TITLE
I found some interfaces not working properly

### DISCUSSION THREAD

**Discussion by CodeIdeal** [realworld-apps/realworld#859] [DEMO / API health status] [answered]
> Test user
> ```json
> {
>     "email": "didi@didi.com",
>     "username": "didi",
>     "token": "eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJlbWFpbCI6ImRpZGlAZGlkaS5jb20iLCJ1c2VybmFtZSI6ImRpZGkiLCJwYXNzd29yZCI6IiQyYSQxMCRnYTJLUzFzbFB4cGdzOHZVZ0MzazIuYVhDbS9qMHpwL0FFR2c5YnZObUlIbnBILnUvSUlLTyIsImJpbyI6bnVsbCwiaW1hZ2UiOiJodHRwczovL2ltZzAuYmFpZHUuY29tL2l0L3U9MjIxNjExOTc3OSw2NTMwNTUzODYmZm09MjUzJmZtdD1hdXRvJmFwcD0xMzgmZj1KUEVHP3c9NTAwJmg9NTAwIiwiaWF0IjoxNjQyNjQ5NDMyLCJleHAiOjE2NDc4MzM0MzJ9.p7zmm-MW6krdIiHWj74C52v6S7md-cz_BYLtMV70ReU"
> }
> ``` 

1. In the user information returned by the login interface, the avatar link incorrectly returns the bio field. 
reproduce curl: 
```shell 
curl -X "POST" "https://api.realworld.io/api/users/login/" \
     -H 'Content-Type: application/json' \
     -H 'Accept: application/json' \
     -d $'{
  "user": {
    "email": "didi@didi.com",
    "password": "123123123"
  }
}'
```

2. The interface to get the list of articles, if you are not logged in you can only get the default three articles, if you are logged in you can only get the default three articles plus your own published articles. Why can't I see other people's posts?
reproduce curl: 
```shell 
# authorization
curl "https://api.realworld.io/api/articles/?limit=10&offset=0" \
     -H 'Accept: application/json'
# unauthorized
curl "https://api.realworld.io/api/articles/?limit=10&offset=0" \
     -H 'Accept: application/json' \
     -H 'authorization: Token eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJlbWFpbCI6ImRpZGlAZGlkaS5jb20iLCJ1c2VybmFtZSI6ImRpZGkiLCJwYXNzd29yZCI6IiQyYSQxMCRnYTJLUzFzbFB4cGdzOHZVZ0MzazIuYVhDbS9qMHpwL0FFR2c5YnZObUlIbnBILnUvSUlLTyIsImJpbyI6bnVsbCwiaW1hZ2UiOiJodHRwczovL2ltZzAuYmFpZHUuY29tL2l0L3U9MjIxNjExOTc3OSw2NTMwNTUzODYmZm09MjUzJmZtdD1hdXRvJmFwcD0xMzgmZj1KUEVHP3c9NTAwJmg9NTAwIiwiaWF0IjoxNjQyNTgxNjQwLCJleHAiOjE2NDc3NjU2NDB9._cS2BHYR2hQmzwGUOijDh9WCeBgsXtAswxX7c5_YapE'
```

**Comment by geromegrignon** [comment, marked as answer]
Hey @CodeIdeal, about not retrieving content from other accounts, it's part of the documentation: https://realworld-docs.netlify.app/docs/specs/frontend-specs/api#api-limitations

We had literally all kinds of undesirable content added by some users last year so we had to add limitations to visibility.

About the image data, I'll fix it.

**Reply by geromegrignon** [reply]
fixed

**Reply by CodeIdeal** [reply]
Got it. Thanks for the quick fix.

**Comment by CodeIdeal** [comment]
@geromegrignon 
There is an another problem. When I want to update only one field, the interface for updating user information returns an error.
```shell
curl -X "PUT" "https://api.realworld.io/api/user/" \
     -H 'Content-Type: application/json' \
     -H 'Accept: application/json' \
     -H 'authorization: Token eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJlbWFpbCI6ImRpZGlAZGlkaS5jb20iLCJ1c2VybmFtZSI6ImRpZGkiLCJwYXNzd29yZCI6IiQyYSQxMCRnYTJLUzFzbFB4cGdzOHZVZ0MzazIuYVhDbS9qMHpwL0FFR2c5YnZObUlIbnBILnUvSUlLTyIsImJpbyI6bnVsbCwiaW1hZ2UiOiJodHRwczovL2ltZzAuYmFpZHUuY29tL2l0L3U9MjIxNjExOTc3OSw2NTMwNTUzODYmZm09MjUzJmZtdD1hdXRvJmFwcD0xMzgmZj1KUEVHP3c9NTAwJmg9NTAwIiwiaWF0IjoxNjQyNTgxNjQwLCJleHAiOjE2NDc3NjU2NDB9._cS2BHYR2hQmzwGUOijDh9WCeBgsXtAswxX7c5_YapE' \
     -d $'{"bio": "test"}'
```

**Reply by geromegrignon** [reply]
Can you give it another try? I made changes.
